translations/texts/objects/apex/apexarmchair/apexarmchair.object.json

```
@@ -20,7 +20,7 @@
     },
     "Texts": {
       "Eng": "Apex fashions have evolved since this chair was made; sadly, Apex freedoms have not.",
-      "Rus": ""
+      "Rus": "С тех пор, как был создан этот стул, мода Апексов развивалась; к сожалению, свободы у Апексов нет."
```

```suggestion
      "Rus": "С тех пор, как был создан этот стул, мода Апексов развивалась, в отличие от их свободы, к сожалению."
```
Тут Present Perfect, потому have не означает наличие или отсутствие чего-либо

---

Да у меня там много чего не корректно написано, но я всё исправлю.